<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Projects</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.2/dist/katex.min.css" integrity="sha384-yFRtMMDnQtDRO8rLpMIKrtPCD5jdktao2TV19YiZYWMDkUR5GQZR/NOVTdquEx1j" crossorigin="anonymous">
		<script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.2/dist/katex.min.js" integrity="sha384-9Nhn55MVVN0/4OFx7EE5kpFBPsEMZxKTCnA+4fqDmg12eCTqGi6+BB2LjY8brQxJ" crossorigin="anonymous"></script>
		<script defer src="https://cdn.jsdelivr.net/npm/katex@0.10.2/dist/contrib/auto-render.min.js" integrity="sha384-kWPLUVMOks5AQFrykwIup5lo0m3iMkkHrD0uJ4H5cjeGihAutqP0yW0J6dpFiVkI" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>
	</head>

	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					
				<!-- Nav -->
					
				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<section class="post">
								<header class="major">
									<h1>STAT 888 Project: High Dimensional Mediational Analysis</h1>
									<p> <b>Abstract:</b> Since its inception by Sewall Green Wright, mediation analysis has faced many criticisms and challenges. To avoid well-known limitations like inflated test statistic in Sobel's test, practitioners have been trying to adopt a stepwise procedure to mediation analysis. But due to the recent surge in using high dimensional biomarkers and omics data, the number of tests prior to the mediation analysis has become so large that we are in need of new theories that can deal with high dimensionality in mediators and exposures. In this project, I'll review both the pioneering and recent articles on high dimensional mediation analysis and share some ideas about where the future research can be directed to.</p>
								</header>
								<div class="image"><img src="pic01.jpg" alt="" /></div>
								<header> <b>Details</b> </header> 
								<p>Recent advances in DNA sequencing technologies have led to a genomic revolution where we now have tens of thousands of known gene disease association discoveries. Some people argue, knowing that a gene causes a disease is not as valuable as knowing how the gene causes the disease. For example, if the genes cause the disease by perhaps affecting some blood biomarkers. Biomarker measuring technologies have also come along a huge development recently so now there is this omics revolution more than just the genomics revolution. In this case, the blood biomarkers are mediators and the genes are exposures, both of which are high dimensional, making any prior test to simplify the composite null hypothesis (note that in a mediation analysis, the "null" hypothesis is composite, unlike other tests) quite impossible (Huang et al. 2015, Huang and Pan 2016). The regular mediation analysis will take the form of graph A above, while the high dimensional mediation analysis is of the form of graph B. </p>
								<p> Here are some challenges for the high dimensional mediation analysis. First, the mediators and often times the exposures are high dimensional and we want to separate the effects through different mediators. Second, the mediators (e.g. biomarkers) can be highly correlated. Third, the biomarkers sometimes have distributions that are not well known. Fourth, we don't know the causal ordering. Fifth, the dependence of the outcome can be difficult to model correctly and the more the biomarkers, the error can propagate. More discussions can be seen in Daniel et al. (2015). </p>
								<p> A counterfactual (or potential outcome) framework can be applied to formally define the causal effects in this case. Then the main effects to estimate would be the average natural direct effect (NDE) and the average natural indirect effect (NIE) (Song et al. 2019). However, we have to make the following assumptions to estimate these effects: 1) no unmeasured exposure-outcome confounding, 2) no unmeasured mediator-outcome confounding given the exposure, 3) no unmeasured exposure-mediator confounding, and 4) no mediator-outcome confounding affected by the exposure.</p>
								<p> In this project, I'll review some of the articles mentioned above and also the following recent articles: Guo et al. (2022): High-Dimensional Mediation Analysis for Selecting DNA Methylation Loci Mediating Childhood Trauma and Cortisol Stress Reactivity, Zhao and Luo (2016): Pathway Lasso: Estimate and Select Sparse Mediation Pathways, and Song et al. (2021): Bayesian hierarchical models for high-dimensional mediation analysis with coordinated selection of correlated mediators.</p>
								
								<header> <b>Notations</b> </header>
                                                                <p> Let \(T\) be the treatment (or exposure) variable, \(Y\) be the outcome, and \(M\) be the mediator. In general, we will let \(M\) stand for the set of all intermediate variables (hence, high dimensional mediation) between \(T\) and \(Y\). We will use the counterfactual notation \(Y_t(u)\) to denote the value that \(Y\) would attain in unit (or situation) \(U = u\) under the control regime \(do(T = t)\). Eq. (4) in Pearl (2000a, Ch. 7) has more formal semantics of these counterfactual expressions. Many concepts associated with direct and indirect effect require comparison to a reference value of \(T\), that is, a value relative to which we measure changes. We will designate this reference value by \(t^*\).</p>
                                                                
								<header> <b>Identification of the estimands</b> </header>
								<p> Natural Direct Effects:</p>
								<p> An event \(T = t\) is said to have a natural direct effect on variable \(Y\) in situation \(U = u\) if the following inequality holds: \(Y_{t^*}(u) \ne Y_{t,M_{t^*}(u)}(u)\). In words, the value of \(Y\) under \(T = t^*\) differs from its value under \(T = t\) even when we keep \(M\) at the same value (\(M_{t^*}(u)\)) that \(M\) attains under \(T = t^*\).We can easily extend this definition from events to variables by defining \(T\) as having a natural direct effect on \(Y\) (in model \(M\) and situation \(U = u\)) if there exist two values, \(t^*\) and \(t\), that satisfy \(Y_{t^*}(u) \ne Y_{t,M_{t^*}(u)}(u)\). Note that this definition does not require that we specify a value \(m\) for \(M\); that value is determined naturally by the model, once we specify \(t\); \(t^*\), and \(u\). If one is interested in the magnitude of the natural direct effect, one can take the difference called the natural direct effect, \(NDE(t, t^*; Y, u)=Y_{t,M_{t^*}(u)}(u)-Y_{t^*}(u)\). Consequently, the average natural direct effect is given by \(NDE(t, t^*; Y, u)=E(Y_{t,M_{t^*}})-E(Y_{t^*})\). As noted in (Robins and Greenland, 1992), we cannot generally evaluate the average natural direct effect from empirical data. But Pearl (2012) showed if there exists a set \(W\) of covariates, nondescendants of \(T\) or \(M\), such that \(Y_{t,m}\) is conditionally independent of \(M_{t^*}\), given \(W\)), then the average natural direct effect is experimentally identifiable, and it is given by: 

								$$NDE(t, t^*; Y) = \sum_{w,m}[E(Y_{t,m}|w)-E(Y_{t^*,m}|w)] P(M_{t^*} = m | w)P(w).$$

								Proof:

								$$E(Y_{t,M_{t^*}}=y)=\sum_{w}\sum_{m}[E(Y_{t,m}=y|M_{t^*}=m,W=w]P(M_{t^*} = m | w=w)P(w=w)$$
								
								Using the fact that \(Y_{t,m}\) is conditionally independent of \(M_{t^*}\), given \(W\),

								$$E(Y_{t,M_{t^*}}=y)=\sum_{w}\sum_{m}[E(Y_{t,m}=y|W=w]P(M_{t^*} = m | w=w)P(w=w)$$

								Each factor in the above equation is identifiable. By law of composition, \(E(Y_{t*})=E(Y_{t^*}M_{t^*})\) and completes the proof.

								The conditional independence relation can easily be verified from the causal graph associated with the model. Using a graphical interpretation of counterfactuals, \(W\) d-separates \(Y\) from \(M\) in the graph formed by deleting all arrows emanating from \(T\) and \(M\). However, The identification of the natural direct effect from nonexperimental data requires stronger conditions. From the above we see that it is sufficient to identify the conditional probabilities of two counterfactuals: \(P(Y_{t,m}=y|W=w)\) and \(P(M_{t^*}=m|W=w)\), where \(W\) is any set of covariates (Pearl 2012).</p>

								<p>Natural Indirect Effects:</p>
								<p>Since the NDE and NIE are connected to each other via \(TE=NDE-NIE_r\) (where, \(TE\) is the total effect and \(NIE_r\) stands for the natural indirect effect under the reverse transition), all our discussions above concerning the identification of NDE should apply to NIE as well (Pearl, 2009b, p. 132; Pearl, 2009a, 2012a).</p> 

							</section>

					</div>

				<!-- Footer -->
					<footer id="footer">
						
						<section class="split contact">
							<section class="alt">
								<h3>Address</h3>
								<p>B315 Medical Sciences Center<br />
								<p>1300 University Avenue<br />	
								Madison WI 53706</p>
							</section>
							<section>
								<h3>Email (for further queries)</h3>
								<p><a href="#">nawan2@wisc.edu</a></p>
							</section>
							
						</section>
					</footer>

				

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
